src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -198,6 +198,12 @@ ManifestCache::configManifest (
         Throw<std::runtime_error> ("Unverifiable manifest in config");
     }
 
+    // Trust our own master public key
+    if (!trusted(m.masterKey) && !unl.trusted (m.masterKey))
+    {
+        addTrustedKey (m.masterKey, "");
+    }
```

Comment:
Am I correct that the only reason we need to trust our own manifest is so that we'll send the current value to our new peers in `PeerImp::doProtocolStart()`?